XTS/localjunit/xtstest/src/org/jboss/jbossts/xts/servicetests/webbean/XTSHTTPServiceTestRunner.java

```diff
@@ -66,12 +67,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
                 }
 
                 try {
-                    _currentTest = (XTSServiceTest)testClass.newInstance();
+                    _currentTest = (XTSServiceTest)testClass.getDeclaredConstructor().newInstance();
                 } catch (InstantiationException ie) {
                     throw new ServletException("XTSHTTPServicetestRunner : cannot instantiate test class " + _testClassName, ie);
                 } catch (IllegalAccessException iae) {
                     throw new ServletException("XTSHTTPServicetestRunner : cannot access constructor for test class " + _testClassName, iae);
-                }
+                } catch (InvocationTargetException e) {
+                    throw new ServletException("XTSHTTPServicetestRunner : cannot access constructor for test class " + _testClassName, e);
+                 } catch (NoSuchMethodException e) {
+                    throw new ServletException("XTSHTTPServicetestRunner : cannot access constructor for test class " + _testClassName, e);
+                 }
```

Comment:
The reported messages should be different than the `IllegalAccessException` case

Reply:
@jmfinelli I am thinking to use the following:

 ```
catch (InvocationTargetException e) {
    throw new ServletException("XTSHTTPServicetestRunner : cannot invoke constructor for test class " + _testClassName, e);
 } catch (NoSuchMethodException e) {
     new ServletException("XTSHTTPServicetestRunner : cannot find constructor for test class " + _testClassName, e);
 }

```
Let me know if this looks good or should i need to change something?

Reply:
It looks great! Thanks @anilabhabaral

Reply:
@tomjenkinson i am new to this project. Could you please help me to understand how to use the localization framework? Did you want me to add this message in a Logger class and use the reference here as log message like `RecoveryLogger.i18NLogger.error_recovery_coordinator_CoordinatorRecoveryInitialisation_4(className, e);` ?

Reply:
No worries, and thank you for the contribution. It will be something like you are saying, I will add an example shortly

Reply:
Sorry, I have realised we are in the tests so it's not needed! Sorry for the noise

Reply:
(and your proposed messages look good to me too - thank you)

Reply:
Thank you @tomjenkinson